.agents/skills/uloop-hot-reload/SKILL.md

```
@@ -72,6 +72,18 @@ initializers, attributes, added or removed members — is reported as a `Warning
 entry as well; without a baseline it stays silent. Either way, use `uloop compile`
 for such edits.
 
+### Tunable values: prefer a getter over a const
+
+`const` edits never take effect through hot reload: C# bakes const values into every
+call site at compile time. When you expect to tune a value while Play Mode is running
+(speeds, amplitudes, sensitivities), expose it as a static property getter instead:
+
+    public static float HeightAmplitude => 5f;
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)